U4-7580 - Renaming document type folders done

lars-erik opened the pull request.

JimBobSquarePants commented:
Bumpity bump.

Shazwazza commented on src/Umbraco.Core/Services/ContentTypeService.cs:
```diff
@@ -81,6 +81,36 @@ public ContentTypeService(IDatabaseUnitOfWorkProvider provider, RepositoryFactor
+
+                    return Attempt.Succeed(new OperationStatus<EntityContainer, OperationStatusType>(container, OperationStatusType.Success, evtMsgs));
+                }
+                catch (Exception ex)
```
What is this catching and why?

lars-erik commented:
Which jquery/bootstrap files? ;)
("Who controls the past," ran the Party slogan, "controls the future: who controls the present controls the past." - Orwell)

Regarding the rest: I rebased and removed the clever comment. I basically copied the pattern of execution/try/catch/attempt from https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/Services/ContentTypeService.cs#L79. It looks like there was a whole lot more Attempt<T> going on back when I wrote it. It should still work. No conflicts. And the TODO was also a copy.
